src/swap/components/SwapProvider.tsx

```diff
@@ -60,7 +65,7 @@ export function SwapProvider({
     },
   }); // Component lifecycle
 
-  // Update lifecycle status, statusData will be persisted for the full lifecycle
+  // Update lifecycle status, statusData will be persisted for the full lifeCycle
```

Comment:
its lifecycle now

Reply:
oops - my cherry pick loaded the old version

pushed a fix